### PR TITLE
admin messages

### PR DESCRIPTION
**Pull request by JarekW**
concept for now,
is it good?

### CODE REVIEW

**Comment by chives** [comment]
concept is ok, but mark this PR with [WIP]

File: Resources/config/services.xml [hunk 1]
```diff
@@ -116,5 +116,25 @@
...
             <tag name="admin.manager.visitor" />
         </service>
+
+        <service id="admin.messages.flash" class="FSi\Bundle\AdminBundle\Message\FlashMessages">
+            <argument type="service">
+                <service class="Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface">
+                    <factory service="session" method="getFlashBag"/>
+                </service>
+            </argument>
+            <argument type="string">fsi_admin</argument>
```
Comment (JarekW) [inline Resources/config/services.xml]: is it needed that this value to be somehow configurable?

File: Resources/translations/FSiAdminBundle.pl.yml [hunk 2]
```diff
@@ -65,3 +65,10 @@ crud:
         title: Podgląd elementu
         button:
             back: Powrót
+
+messages:
+  form:
+    save: Dane zostały zapisane.
+    error: Formularz został wypełniony nieprawidłowo.
+  batch:
+    success: Operacja została wykonana.
```
Comment (JarekW) [inline Resources/translations/FSiAdminBundle.pl.yml]: maybe you will have some better propositions for this messages

**Comment by JarekW** [comment]
@fsi-open ?

**Comment by JarekW** [comment]
@fsi-open

File: Resources/config/services.xml (hunk 1 above)
Comment (szymach) [inline Resources/config/services.xml]: I doubt it

File: Resources/translations/FSiAdminBundle.pl.yml (hunk 2 above)
Comment (szymach) [inline Resources/translations/FSiAdminBundle.pl.yml]: Perhaps something in this manner: 
save: Dane zostały poprawnie zapisane.
error: Nastąpił błąd podczas zapisu danych z formularza.
success: Operacja zakończona powodzeniem.

Comment (JarekW) [inline Resources/translations/FSiAdminBundle.pl.yml]: >error: Nastąpił błąd podczas zapisu danych z formularza.
this suggest that something is wrong inside app

**Comment by JarekW** [comment]
@fsi-open something else will need to be changed?

File: Resources/translations/FSiAdminBundle.pl.yml (hunk 2 above)
Comment (szymach) [inline Resources/translations/FSiAdminBundle.pl.yml]: well the key *is* called `error` :P

**Comment by JarekW** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Comment by chives** [comment]
@fsi-open

**Comment by norzechowicz** [comment]
:+1:

File: Resources/views/base.html.twig [hunk 3]
```diff
@@ -56,7 +56,16 @@
 <div class="container">
     <div class="row">
         <div class="col-lg-12">
-        {% block above_content %}{% endblock above_content %}
+        {% block above_content %}
+            {% set typeClass = {success:'success',error:'danger',info:'info'} %}
```
Comment (szymach) [inline Resources/views/base.html.twig]: arguments not separated by spaces

**Comment by szymach** [comment]
otherwise :+1:

**Comment by rn0** [comment]
:+1:

**Pull request merged**